integration-libs/opf/cta/root/model/opf-cta.model.ts

```diff
@@ -33,9 +33,7 @@ export interface CtaProductItem {
 export enum CtaScriptsLocation {
   CART_MESSAGING = 'CART_MESSAGING',
   PDP_MESSAGING = 'PDP_MESSAGING',
-  PDP_QUICK_BUY = 'PDP_QUICK_BUY',
```

Comment:
not used and causing compile error if leaving them.
We'll add them when we'll need it.